.rubocop.yml

```diff
@@ -80,6 +80,9 @@ Style/IdenticalConditionalBranches:
 Style/IfInsideElse:
   Enabled: false
 
+Style/LeadingCommentSpace:
+  Enabled: false
```

Comment:
Not sure what you mean. The "code" itself doesn't need documentation (I presume). And if people want to understand why that code is present, the PR serves that purpose, not comments in the code.